grandnode/grandnode
# List categories in selected language

Type: **feature**

## Issue
When administrator select category for product, more comfort see categories in selected(work) language.

## Solution
Use selected language id from work context when retrieve available categories list.

## Breaking changes
No.

## Testing
1. Add some language(more then one in total)
2. Add translation for some category
3. In product create\update, on mapping tab try add or update category - see categories list in selected language
